### ISSUE TITLE
DreamerV4: implement Attention Logits Soft Clipping

### ISSUE THREAD

**Issue by bschink** [Arian-Urdu/pokemon-rl#5]
- nicklashansen's Pytorch implementation doesn't use Attention Logits Soft Clipping
- we should implement it because the original DreamerV4 paper uses it
